### DISCUSSION TITLE
Can I use this tool on my ESP-32?

### DISCUSSION THREAD

**Discussion by 0xShakhawat** [spacehuhn/esp8266_deauther#1634] [Q&A]
It's written for ESP-8266. ESP32 and ESP8266 have some differences in architecture. 🫡
I am in doubt whether it will work in ESP-32 or not. Or, This tool needs to be modified to work with the ESP32? 🤔

#0xShakhawat

**Comment by Mrxlover** [comment]
It Won't work....trust me....

**Reply by 0xShakhawat** [reply]
Which tool should I use? ...

**Comment by Mrxlover** [comment]
You can use ESP 367

**Comment by Wemers0n** [comment]
probably not, because the architecture is different
